include/LightGBM/config.h

```diff
@@ -569,7 +571,7 @@ struct Config {
   // check = >= 0.0
   // desc = controls smoothing applied to tree nodes
   // desc = helps prevent overfitting on leaves with few samples
-  // desc = if set to zero, no smoothing is applied
+  // desc = if ``0.0`` (the default), no smoothing is applied
```

Comment:
Similarly to [`monotone_penalty`](https://github.com/microsoft/LightGBM/blob/master/docs/Parameters.rst#data), [`feature_pre_filter`](https://github.com/microsoft/LightGBM/blob/master/docs/Parameters.rst#feature_pre_filter) and [`predict_disable_shape_check`](https://github.com/microsoft/LightGBM/blob/master/docs/Parameters.rst#predict_disable_shape_check)